test/metabase/models/collection_test.clj

```diff
@@ -1681,7 +1681,7 @@
 
 (deftest instance-analytics-collections-test
   (testing "Instance analytics and it's contents isn't writable, even for admins."
-    (t2.with-temp/with-temp [Collection audit-collection {}
+    (t2.with-temp/with-temp [Collection audit-collection {:id 13371337 :type "instance-analytics"}
```

Comment:
you should use `perms/audit-db-id` rather than the name here. Will be quite mystifying in the future

Reply:
```suggestion
    (t2.with-temp/with-temp [Collection audit-collection {:id perms/audit-db-id :type "instance-analytics"}
```